src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +17,25 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double objectCurrent;
+    double inCurrent=0;
+    double changeTime;
+    double inVoltage;
+    double Voltage;
+    QTime initialTime;
+    QTime currentTime;
+    double AmpHours;
+    double Current;
+    int counter;
+    double SumCurrent=0;
+    double TimeLeft;
+    double AverageCurrent;
+    double Hours;
+    double Minutes;
+    double Seconds;
+    double Milliseconds;
+    QTime t;
+    double AmpChange;
+    double dTime;
```

Comment:
So you have a fair amount of variables which are not being used, you should remove them to keep this file as brief as possible.